torch/_torch_docs.py

```diff
@@ -3178,6 +3178,54 @@ def merge_dicts(*dicts):
     tensor([ 0.,  2.,  1.,  0.])
 """.format(**common_args))
 
+add_docstr(torch.histogram,
+           r"""
+histogram(input, bins=10, weights=None, min=0, max=0, density=False) -> Tensor
+
+Computes the histogram of a tensor.
+
+Args:
+    {input}
+    bins (int or 1D Tensor): If int, defines the number of bins. If Tensor,
```

Comment:
lowercase "tensor" and end the description of each param with a period.

Reply:
Done.